src/Source/WMSSource.js

```diff
@@ -0,0 +1,86 @@
+import Source, { supportedFormats } from './Source';
+import URLBuilder from '../Provider/URLBuilder';
+
+class WMSSource extends Source {
+    /**
+     * Images source
+     * @constructor
+     * @extends Source
+     * @param {sourceParams}  source
+     * @param {string} source.name name of layer wms
+     * @param {Extent} source.extent extent of wms source
+     * @param {string} [source.style=''] style of layer wms
+     * @param {number} [source.heightMapWidth=256] size texture in pixel
+     * @param {string} [source.version='1.3.0'] wms version
+     * @param {string} [source.axisOrder] wms axis order ('wsen' or 'swne')
+     * @param {boolean} [source.transparent=false] source return texture with transparence
+     * @param {Object} [source.zoom]
+     * @param {number} [source.zoom.min] layer's zoom minimum
+     * @param {number} [source.zoom.max] layer's zoom maximum
+    */
+    constructor(source) {
+        super(source);
+        if (!source.name) {
+            throw new Error('source.name is required.');
+        }
+
+        if (!source.extent) {
+            throw new Error('source.extent is required');
+        }
+
+        if (!source.projection) {
+            throw new Error('source.projection is required');
+        }
+
+        this.name = source.name;
+        this.zoom = source.zoom || { min: 0, max: 21 };
+
+        if (!supportedFormats.includes(source.format)) {
+            throw new Error(`Layer ${source.name}: unsupported format '${source.format}', should be one of '${supportedFormats.join('\', \'')}'`);
+        }
+
+        this.format = this.format || 'image/png';
+        this.style = source.style || '';
+        this.width = source.heightMapWidth || 256;
+        this.version = source.version || '1.3.0';
+        this.transparent = source.transparent || false;
+
+        if (!source.axisOrder) {
+        // 4326 (lat/long) axis order depends on the WMS version used
+            if (source.projection == 'EPSG:4326') {
+            // EPSG 4326 x = lat, long = y
+            // version 1.1.0 long/lat while version 1.3.0 mandates xy (so lat,long)
+                this.axisOrder = (this.version === '1.1.0' ? 'wsen' : 'swne');
+            } else {
+            // xy,xy order
+                this.axisOrder = 'wsen';
+            }
+        }
+
+        let crsPropName = 'SRS';
```

Comment:
Why not directly `const crsPropName = (this.version === '1.3.0') ? 'CRS' : 'SRS';` ?